### DISCUSSION TITLE
Multiple Firestore Databases

### DISCUSSION THREAD

**Discussion by GameOfAndroids** [GitLiveApp/firebase-kotlin-sdk#664] [Ideas]
Hello all!
I am using a Firebase project that has multiple Firestore databases. I never found any documentation or api supporting multiple Firestore databases. For our use case, unfortunately we cannot move forward with a Multiplatform Project without this functionality. Any word if this is in the pipeline?
Thanks!
Tim

**Comment by nbransby** [comment]
Can you link to the API you need from the android sdk docs?
